### PR TITLE
now with namedtupes

### PR DESCRIPTION
**Pull request by shaneknapp**
...and actual handling of folks who have left the org, but have spends in the billing CSV

### CODE REVIEW

File: aws-audit/aws-audit.py [hunk 1]
```diff
@@ -302,7 +302,7 @@ def generate_simple_report(user_dict, limit, display_ids):
            '== Current AWS totals:  $%s USD (only shown below: > $%s) ==\n\n' \
            % (sum_str, limit)
 
-  for acct in sorted(account_details, key = lambda acct: -acct[2]):
+  for acct in sorted(account_details, key = lambda acct: acct[2], reverse = True):
```
Comment (shaneknapp) [inline aws-audit/aws-audit.py]: thanks, documentation!  https://docs.python.org/3.5/howto/sorting.html#ascending-and-descending

**Comment by shaneknapp** [comment]
@jey any comments?  things seem to be working great (see the email that came out this morning @ 8am), and i'd like to commit this and get working on some general cleanup and refactoring so i can open-source this.

**Comment by jey** [comment]
Looks good to me! Only comment is that "only shown below: > $5.0" is
currently a lie; even $0.00 line items are being included in the report

**Comment by shaneknapp** [comment]
@jey re: only shown below....  really?  the limit is specified w/the --limit option, and defaults to $5.00.  looking at the latest report, i don't see any spends being shown that are less that $5.

**Comment by shaneknapp** [comment]
alright, merging.

**Pull request merged**